default/python/AI/PlanetUtilsAI.py

```diff
@@ -43,7 +41,7 @@ def get_capital():
     universe = fo.getUniverse()
     empire = fo.getEmpire()
     if empire is None:
-        print >> sys.stderr, "Danger Danger! FO can't find an empire for me!!!!"
+        warn("Danger Danger! FO can't find an empire for me!!!!")
```

Comment:
Danger Danger, but not an error?

Reply:
Not even sure why this part of the code exists, this case is handled as soon as the AI code is called. Even if not, we would have a gazillion of errors thrown if an empire does not exist as it is nowhere else in the code.

Probably best to remove that logging completely...